Sources/PitchSpeller/Wetherfield/EdgeCapacity.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  EdgeCapacity.swift
+//  NotationModel
+//
+//  Created by Ben Wetherfield on 06/16/18.
+//
+
+enum EdgeCapacity:
+    Comparable
+{
+    case real(Float)
+    case bigM
+}
+
+// MARK: - Comparable
+// bigM edges always 'win' comparisons - otherwise like a regular comparison
+func < (lhs: EdgeCapacity, rhs: EdgeCapacity) -> Bool {
```

Comment:
This might be ever so slightly more elegant:

```Swift
switch (lhs, rhs) {
case let (.real(lhs), .real(rhs)):
    return lhs < rhs
case let (.bigM, .real):
    return false
case let (.real, .bigM):
    return true
}
```

Reply:
Totally! I may save this change for a different branch / PR... still figuring out what that would be (potentially intedges). As it stands, building this enum on top of `Float` isn't that useful because `Float.infinity` does the same thing as we discussed elsewhere